mobile/src/data/userMockData.js

```diff
@@ -0,0 +1,8 @@
+const user = {
+  name: 'Stephanie Jones',
+  status: 'active',
```

Comment:
(general question) on the video that you show, are you manually changing the status of the mock data to see different style of each status?

Reply:
Yes. I manually changed the data to showcase different styles.